src/App.vue

```diff
@@ -820,58 +774,51 @@ body {
 
 .gamebox .gamebox-bottom {
   cursor: default;
-  position: relative;
-  margin-top: -21%;
-  width: 100%;
-  height: 15%;
+
+  height: 40px;
   position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
   background-color: rgba(var(--all-color-front), 1);
   border-radius: 0px 0px 10px 10px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 0.5rem;
 }
 
 .gamebox .gamebox-bottom span {
   font-family: Nunito-Bold;
-  position: absolute;
-  margin-top: 14px;
-  margin-left: 10px;
+
   font-size: 12px;
   color: #dadada;
-  display: block;
+  flex-grow: 1;
 }
 
 .gamebox .gamebox-bottom .menu {
-  position: absolute;
-
-  margin-left: 165px;
-  margin-top: 13px;
-  display: block;
   content: url("./assets/svg/menu.svg");
   height: 18px;
   width: 18px;
   cursor: pointer;
-  cursor: default;
+  margin-left: 0.5rem;
 }
 
 .mainPageGamesList {
```

Comment:
The distance between the title and the box is too much. Consider decreasing it:

![image](https://github.com/Lazap-Development/Lazap/assets/59381835/7770b987-b070-4ae0-91cf-8f7643003402)
![image](https://github.com/Lazap-Development/Lazap/assets/59381835/5af4330b-0903-4bfa-b9b7-e20defde0be9)

Reply:
reduced the bottom margin for the recently launched text to make the games appear more in the middle.